File: src/main/webapp/src/app/core/models/training/training.participants.ts
import {Training} from "./training";

export class TrainingParticipants {
    id: number;
    training: Training;
    participants: [{Participant}];

    constructor(t) {
        {
            this.id = t.id;
            this.training = t.training;
            this.participants = t.participants;
        }
    }
}

File: src/main/java/org/charess/training/domain/training/TrainingParticipants.java
package org.charess.training.domain.training;

import java.util.List;

public class TrainingParticipants {

    private Training training;
    private List<Participant> participants;

    public Training getTraining() {
        return training;
    }

    public void setTraining(Training training) {
        this.training = training;
    }

    public List<Participant> getParticipants() {
        return participants;
    }

    public void setParticipants(List<Participant> participants) {
        this.participants = participants;
    }
}
